website/docs/r/service_aws.html.markdown

```diff
@@ -12,6 +12,16 @@ This resource allows you to manage a service account that is tied to an AWS IAM
 
 The native AWS identity provider uses a combination of AWS IAM and AWS KMS to provide access to SecretHub for any service running on AWS (e.g. EC2, Lambda or ECS) without needing a SecretHub credential.
 
+## Example Usage
+
+```terraform
+resource "secrethub_service_aws" "demo_service_account" {
+  repo        = "workspace/repo"
+  role        = "${aws_iam_role.your_application.name}"
```

Comment:
If we're calling the IAM role `your_application`, do we want to call the service so as well?

Reply:
And maybe for the repo name as well?

Reply:
> If we're calling the IAM role your_application, do we want to call the service so as well?

Yes.

> And maybe for the repo name as well?

No, will only cause confusion.